lib/net/peerpool.js

```diff
@@ -185,6 +188,29 @@ class PeerPool extends EventEmitter {
       }
     }
   }
+
+  /**
+   * Peer pool status check on a repeated interval
+   */
+  async _statusCheck () {
+    if (this.size === 0) {
+      this.noPeerPeriods += 1
+      if (this.noPeerPeriods >= 3) {
+        const promises = this.servers.map(async server => {
```

Comment:
@ryanio Thank you
I don not know this context in detail yet.
Which do you think is better, [Promise.allSettled](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled) or [Promise.all](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all)?

Reply:
It's okay if it looks good as it is

Reply:
[This too](https://github.com/ethereumjs/ethereumjs-client/pull/133/files#diff-60e55bc916018e3ab309401fa746746bR112)

Reply:
@ryanio let me know if I need to update anything here or you can alternatively also directly push yourself

@tomonari-t thanks for reviewing!

Reply:
@tomonari-t ah thanks. I think `Promise.all` looks good since `Promise.allSettled` seems like it's useful if we need to use the results of the outcomes. In these two cases I think we're just interested in waiting for them to finish.

Reply:
@ryanio I got it! so LGTM 👍